.gitignore

```diff
@@ -35,3 +35,4 @@ build
 builds
 epipe*
 cmake-build-debug
+build.sh
```

Comment:
You shouldn't add build.sh to git ignore

Reply:
fixed